### DISCUSSION TITLE
Welcome to skyshare Discussions!

### DISCUSSION THREAD

**Discussion by nekono-dev** [nekono-dev/skyshare#35] [Announcements]
## 👋 Welcome! 
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
Welcome others and are open-minded. Remember that this is a community we build together 💪.
 
## ようこそ！🙌
コミュニティ外のメンバーとディスカッションをするために`Disscussion`の機能を展開しました。 以下のような場合に活用できます。
  * 疑問に思っていることを質問する。
  * アイデアを共有する。
  * 他のコミュニティメンバーと交流する。

この場はオープンな場所です。コミュニティ外の方々を歓迎します！コミュニティは我々一人一人の手で作り上げられるということを忘れずに！🔥

**Comment by nekono-dev** [comment]
## Developper rules defined!😁

For when increase of Collaborator, I defined Developer's rules!
I provide yous part of access preview environment.
Please read if you join developing

[Developer Reference](https://github.com/nkte8/skyshare/wiki/Developer-Reference-%7C-開発者リファレンス)

## デベロッパールールを制定しました！✌️

Collaboratorが今後増えた時のために、デベロッパールールを制定しました！
Cloudflare Pagesのプレビュー環境の一部のアクセス権限を提供します。
開発される方はお読みください。

[開発者リファレンス](https://github.com/nkte8/skyshare/wiki/Developer-Reference-%7C-開発者リファレンス)

**Comment by cranberri-inari** [comment]
Hi there!! Thank you so much for this tool, this is already incredibly helpful for me posting my artwork more safely! I had a suggestion for a tool for Skyshare. Would it be possible to have the option to upload a custom cropped version of the art for the link image? I'd love to be able to show a different preview image for X to get people excited to click on the link to view the art on Skyshare and Bluesky. Just an idea, thank you for all your hard work on this, excited to see this developed further!
